.github/CODEOWNERS

```diff
@@ -1 +1 @@
-* @simonr0204
+* @braqzen @bitzoic
```

Comment:
```suggestion
* @nfurfaro 
```

Reply:
https://github.com/FuelLabs/sway-applications/pull/619#discussion_r1161747499

Reply:
If anyone else on team @FuelLabs/application-dev want notifications for the repo, I suggest we make the team the codeowner here as well (same as the app-dev repo).

Reply:
I've only re-added myself in the apps repo because I didn't want others to get spammed but then the whole thing happened so I've suggested the same change here.
Either way, now that people should know what that file does I'm happy to be the only one added to it in both repos if they do not want to get spammed with review notifications by default.